Userland/Libraries/LibJS/Console.cpp

```diff
@@ -90,17 +90,56 @@ Value Console::trace()
     return js_undefined();
 }
 
-Value Console::count()
+// 1.2.1. count(label), https://console.spec.whatwg.org/#count
+ThrowCompletionOr<Value> Console::count()
 {
+    auto label = vm().argument_count() ? TRY(vm().argument(0).to_string(global_object())) : "default";
```

Comment:
~Maybe clarify where this default value comes from? In the spec text I don't see this anywhere.~

Oh nvm I see it here https://console.spec.whatwg.org/#ref-for-count. 
Oh boy the web specs are not quite as good the ecma spec I'm used to.
Maybe to clarify things adding:
 `undefined count(optional DOMString label  = "default");` 
somewhere in the method to clarify both the argument parsing and return values?

Personally I would like to see those added as it clarifies some things for me however it isn't crazy to return `undefined` when nothing is happening so up to you

Reply:
I settled on adding this:
```c++
// NOTE: "default" is the default value in the IDL. https://console.spec.whatwg.org/#ref-for-count
auto label = vm().argument_count() ? TRY(vm().argument(0).to_string(global_object())) : "default";
```

Thanks for bringing this up, since I wasn't supplying that default value for the time functions!